src/polyfills/EventEmitter.js

```diff
@@ -0,0 +1,85 @@
+class EventEmitter {
+  constructor() {
+    this.subscriptions = new Map();
+  }
+  subscribe(eventName, callback) {
+    if (!this.subscriptions.has(eventName)) {
+      this.subscriptions.set(eventName, new Set());
+    }
+    const subscription = this.subscriptions.get(eventName);
+    const callbackObj = { callback };
+    subscription.add(callbackObj);
+
+    return {
+      release: () => {
+        subscription.delete(callbackObj);
+        if (subscription.size === 0) {
+          this.subscriptions.delete(eventName);
+        }
+      },
+    };
+  }
+  emit(eventName, ...args) {
+    const subscription = this.subscriptions.get(eventName);
+    if (subscription) {
+      subscription.forEach((subs) => subs.callback.apply(null, args));
+    }
+  }
+}
+
+export class TaskManager extends EventEmitter {
```

Comment:
Remove `export` from this line. We're using `TaskManager` only within this file.

Reply:
thanks for comment @mandy8055_be. I have factored in the changes mentioned by you. So resolving this comment.